test/task_list/filter_test.rb

```diff
@@ -51,10 +51,10 @@ def test_populates_result_with_task_list_items
     incomplete, complete = result[:task_list_items]
 
     assert incomplete
-    assert !incomplete.complete?
+    assert_equal incomplete.complete?, false
 
     assert complete
-    assert complete.complete?
+    assert_equal complete.complete?, true
```

Comment:
Not sure this makes the test better. I'm fine leaving it as it was, it doesn't reduce the quality of the change.